px-context-browser-filter.html

```diff
@@ -32,6 +33,10 @@
     Polymer({
       is: 'px-context-browser-filter',
 
+      behaviors: [
+        Polymer.AppLocalizeBehavior
```

Comment:
Instead of including the localization behavior in here, let's just expose a new property called `placeholder`. Then in `px-context-browser.html` you can call the localization function:

```
 <px-context-browser-filter slot="filter" active$="[[showFilter]]" filter="{{filter}}" filtering="{{filtering}}" placeholder="[[localize('Filter')]]"></px-context-browser-filter>
```